src/api.ts

```diff
@@ -248,9 +251,8 @@ export class MirrordAPI {
   * `VerifiedConfig`.
   */
   async verifyConfig(configPath: vscode.Uri | null): Promise<VerifiedConfig | undefined> {
-    const args = ['verify-config'];
     if (configPath) {
-      args.push(configPath.path);
+      const args = ['verify-config', '--ide', '--path', `${configPath.path}`];
```

Comment:
We're reverting the addition of the `--path` arg, the extension shouldn't use it.